web_src/css/base.css

```
@@ -34,6 +34,8 @@
   /* z-index */
   --z-index-modal: 1001; /* modal dialog, hard-coded from Fomantic modal.css */
   --z-index-toast: 1002; /* should be larger than modal */
+
+  --font-size-label: 14px;
```

```suggestion
  --font-size-label: 12px;
```

14px is too big imho and also does not match GitHub which also uses 12px. Let's keep it at 12px like before.